risc0/zkp/src/core/sha.rs

```diff
@@ -26,20 +26,28 @@ use hex::{FromHex, FromHexError};
 use risc0_zeroio::{Deserialize as ZeroioDeserialize, Serialize as ZeroioSerialize};
 use serde::{Deserialize, Serialize};
 
+/// The size of a word used with the SHA-256 implementation and within
+/// a [Digest] and [Block] representations (32-bits = 4 bytes).
+pub const WORD_SIZE: usize = mem::size_of::<u32>();
```

Comment:
BTW, I think this is already defined in the platform crate, I've been meaning to go thru and reduce duplication of constants.

Reply:
Ah yes, I see that. I'll go ahead and de-duplicate that.